src/Admin/BlockAdmin.php

```diff
@@ -95,10 +95,6 @@ protected function configureRoutes(RouteCollectionInterface $collection): void
         $collection->add('save_position', 'save-position');
         $collection->add('switch_parent', 'switch-parent');
         $collection->add('compose_preview', $this->getRouterIdParameter().'/compose-preview');
-
-        if (!$this->isChild()) {
-            $collection->remove('create');
```

Comment:
If we remove the create route, no new rows can be added on the collection. (the add button does not show).